### ISSUE TITLE
Comparison with EDVR. REDS dataset.

### ISSUE THREAD

**Issue by DenisDiachkov** [csbhr/CDVD-TSP#8]
Hello,

In your paper you compare your model with EDVR. 
Did you train EDVR on your dataset or only tested the pretrained model? In EDVR paper they use REDS dataset. For the sake of fare comparison could you please train your model with REDS dataset? It would be really good to see the CDVD-TSP and EDVR trained on same data.

I think everyone would highly appreciate it.

**Comment by csbhr** [comment]
The EDVR's quantitative results are achieved by using its provided pre-trained model.

Our results on REDS dataset did not exceed EDVR, but we noticed that EDVR is overfitted on REDS dataset and it does not perform well on real datasets.

**Comment by DenisDiachkov** [comment]
Thank you very much for fast reply! It is now much more clear to me. But I still have few questions:

1) So you mean that your model is not overfitted to your "real datasets"? And why REDS is not real one? It is created by the same person (Seungjun Nah) who created GoPro dataset.
2) Did you train your model on REDS or just tested the pretrained on another dataset one?
3) Could you please share the PSNR and SSIM which you got on REDS?
4) You compared your results to SRN-Deblur in your paper. The PSNR value given for SRN is almost the same as the original authors provide (30.26 vs 30.29). However, SSIM values were clearly different (0.9342 vs 0.9014). Does it happen because of different SSIM estimation (because Pytorch doesn't provide official SSIM function)? Did you train SRN model, or just tested the pretrained model?

**Comment by csbhr** [comment]
The "real dataset" mentioned above is the dataset from "Sunghyun Cho, Jue Wang, and Seungyong Lee. Video deblurring for hand-held cameras using patch-based synthesis. ACM TOG, 31(4):64:1–64:9, 2012."
We noticed that the EDVR's deblurring results on the "real dataset" generated by using its provided pre-trained model are not well, that it seems to be overfitting on the REDS dataset.

We have not trained our model to convergence on REDS, which trained about 100epoch. The PSNR/SSIM we got is 32.080/0.9205. You also can train our model on the REDS dataset using the provided codes.

The SRN's quantitative results are achieved by using its provided pre-trained model. We use the PSNR/SSIM calculating function from EDVR's code.
